packages/@adobe/spectrum-css-temp/components/tabs/index.css

```diff
@@ -31,6 +45,24 @@ governing permissions and limitations under the License.
   vertical-align: top;
 
   outline: none;
+
+  &.spectrum-Tabs--dropdown {
+    height: calc(100% - 2px);
+    inset-inline-start: 0;
+    position: absolute;
+    transform: translateY(-50%);
+    top: 50%;
+    width: 100%;
+
+    /* TODO: Ask what to do about the below styles. Need to get them propagated to the Picker's FieldButton/Picker */
+    .spectrum-FieldButton {
+      padding: 0 4px;
+    }
+
+    .focus-ring {
+      border-radius: 2px;
+    }
```

Comment:
As noted, not entirely sure what to do about these styles since they should be affecting Picker's FieldButton.